terraform/aws/dag

```diff
@@ -245,6 +246,25 @@ Terraform commands:
     grafana \t\t\t Open grafana
 EOF)
 
+POSITIONAL=()
```

Comment:
Can you write a line in a README? `terraform/aws/README.md`

Reply:
Yes, also you reminded about 2 other things:
1. changelog
2. the `usage` docs inside of the dag script

Reply:
btw - we don't mark changes in changelog if it's our tooling only and it does not affect nodes

Reply:
Ok, good to know that.